namespace/namespace.go

```diff
@@ -91,73 +91,27 @@ func strip(prefix, key []byte) []byte {
 
 // Iterator creates a new iterator instance
 func (nstore *namespaceStore) Iterator(ctx context.Context, opts corekv.IterOptions) corekv.Iterator {
-	// make a copy of the namespace so that we can safely mutate it within this function
-	namespace := cp(nstore.namespace)
-
-	var hasStart bool
-	var hasEnd bool
-	var hasPrefix bool
 	// we can use unsafe here since we already aquired locks
 	// either prefix (priority) or start/end
```

Comment:
todo: This comment can be removed as we no longer use unsafe.

Reply:
I don't think we used `unsafe` at the time we removed the mutexes anyway :)

Nice spot, thanks Fred.

- [ ] Remove outdated comment